www/js/survey/multilabel/multi-label-ui.js

```diff
@@ -165,6 +165,29 @@ angular.module('emission.survey.multilabel.buttons',
       }
   };
 
+  var expandInputsIfNecessary = function(inputType, inputValue) {
+    console.log("Experimenting with expanding inputs for type "+inputType+" and value "+inputValue);
```

Comment:
Is this console log still needed? It looks like it may be related to your debugging while implementing the change

Reply:
It is still useful to see what the values are, so that we can debug the behavior.
Note that this code was substantially refactored in subsequent PRs:
https://github.com/e-mission/e-mission-phone/pull/876
and
https://github.com/e-mission/e-mission-phone/pull/878